src/notifications/registry.ts

```diff
@@ -0,0 +1,12 @@
+import { storeTrackingOption } from '../analytics/store-tracking-option'
+
+export const actionRegistry = {
```

Comment:
RE a reply to my suggestion to move these different action types to their own constants for re-use around the notifications codebase (comment has been lost in the changes):

> If we will do this, then when we will add the new notifications, we have to modify the constant at many places, that does not look a good idea.

This is one of the main points. When we make the new notifications with actions, we need to specify the action types. To avoid writing string literals and possible typos all over the place (although with proper typings TS can catch string literal typos - most of the UI code is not in TS though), please re-use the same constants. 
(they should never need to be modified as well; please let me know if there's any confusion)